hack/jenkins/minikube_cross_build_and_upload.sh

```diff
@@ -52,3 +52,13 @@ rm -r out/buildroot || true
 
 # Upload everything we built to Cloud Storage.
 gsutil -m cp -r out/* gs://minikube-builds/${ghprbPullId}/
+
+# Upload containers for the PR:
+make localkube-image
+gcloud docker -- push gcr.io/k8s-minikube/localkube-image:$ghprbActualCommit
```

Comment:
It looks like the tag we use is based on the localkube version, not on the commit ID:
https://github.com/kubernetes/minikube/blob/master/Makefile#L45

Reply:
I am planning on using a COMMIT based tag for the jenkins CI testing and a LOCALKUBE_VERSION tag for releases.  When building the image for CI I set the TAG=COMMIT